lib/utils/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
             },
             __PREFIX_LINKS__: program.prefixLinks,
           }),
+          new ExtractTextPlugin('styles.css'),
```

Comment:
Seems reasonable. And since this is a new experimental change that doesn't conflict with anything (unless someone named their css file *.modules.css) we can ship something we're not 100% on and refine it after the fact.